opencti-platform/opencti-front/src/private/components/analyses/malware_analyses/Root.tsx

```diff
@@ -88,6 +88,7 @@ const RootMalwareAnalysis = () => {
               return (
                 <>
                   <StixDomainObjectHeader
+                    entityType={'Malware-Analysis'}
```

Comment:
this one was missing

Reply:
indeed the bug happens even in prod for malware analysis